### PR TITLE
Plant display

### LINKED ISSUE #142: Display plants added to gardens on garden show page

### PR DESCRIPTION
**Pull request by AngelaGuardia**
Displays plants on the garden show page

Fixes #142

### CODE REVIEW

File: app/services/plant_service.rb [hunk 1]
```diff
@@ -10,6 +10,10 @@ def self.add_plant_to_garden(params)
     end
   end
 
+  def self.all_plants_for_garden(params)
+    get_parsed_json("/api/v1/gardens/#{params[:id]}/plants")
+  end
+
```
Comment (D-Lessenden) [inline app/services/plant_service.rb:16]: Great job abstracting this away from the controller

File: spec/features/sensors/new_spec.rb [hunk 2]
```diff
@@ -91,7 +91,7 @@
 
       visit "/gardens/#{@garden.id}"
 
-      expect(page.all(".sensor").size).to eq(6)
+      expect(page.all(".sensor").size).to eq(7)
```
Comment (D-Lessenden) [inline spec/features/sensors/new_spec.rb]: This one has been getting me as well

**Review by lkriffell — APPROVED** [review]
This is awesome! I was wondering where this functionality was

**Review by D-Lessenden — APPROVED** [review]
I pulled this branch and locally I am receiving a strange error:

Psych::SyntaxError:
       (<unknown>): could not find expected ':' while scanning a simple key at line 69 column 1

I believe that this is a local problem. If you are not receiving this error on your local I will be happy to merge.

**Pull request merged**